man/src/image/tag.md

```diff
@@ -9,15 +9,16 @@ entire image name including the optional `TAG` after the ':'.
    present, it may optionally be followed by a port number in the format 
    `:8080`. If not present, the command uses Docker's public registry located at
    `registry-1.docker.io` by default. Name components may contain lowercase 
-   characters, digits and separators. A separator is defined as a period, one or 
+   letters, digits and separators. A separator is defined as a period, one or
    two underscores, or one or more dashes. A name component may not start or end 
```

Comment:
Well, that is how it is done in the 1.1 version of the spec https://github.com/docker/docker/blob/master/image/spec/v1.1.md

I'm going to change back to dash. When specifying ASCII it is clear, and the word dash is used all over the code base.